Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN git clone https://github.com/openlink/virtuoso-opensource.git \
         && cd .. \
         && rm -r /virtuoso-opensource
 
+HEALTHCHECK CMD wget -q -O - http://localhost:8890/sparql'?query=ASK%20%7B%7D' || exit 1
```

Comment:
perhaps this can be improved by checking for a specific triple that should be present? We could add this triple into a "system" graph once the startup script is done. Hence the port change is no longer required